raiden_contracts/tests/test_transaction.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import Mock
+
+import pytest
+
+from raiden_contracts.utils.transaction import check_successful_tx
+
+
+def test_check_successful_tx_with_status_zero():
+    web3_mock = Mock()
+    web3_mock.eth = Mock()
+    web3_mock.eth.getTransactionReceipt = Mock(return_value={'blockNumber': 300, 'status': 0})
```

Comment:
This can also be written as
```
web3_mock.eth.getTransactionReceipt.return_value = {'blockNumber': 300, 'status': 0}
```
which is a bit nicer IMO.

Reply:
Will do this in this PR.